pkg/api/platformapi/allocatorapi/vacate_responses_test.go

```diff
@@ -638,6 +638,12 @@ func newAPMVacateMove(t *testing.T, alloc string, move vacateCaseClusterConfig,
 				"allocator_down": {"false"},
 				"validate_only":  {"true"},
 			},
+			Body: mock.NewStringBody(
```

Comment:
perhaps it would be easier to use `mock.NewStructBody(models.MoveClustersRequest{})` and use the appropriate type populating only the vacate type, here it would be `ApmClusters`, and so on.

It would make the helper test code more durable here since it will grow organically as we add more workload types (Agent / Fleet), and we won't need to update this string payload.